doc/source/download.rst

```diff
@@ -321,9 +321,9 @@ the ``mamba`` package manager.
 ::
 
     conda update -n base -c conda-forge conda
+    conda install -n base --override-channels -c conda-forge mamba 'python_abi=*=*cp*'
```

Comment:
the sentence above that block should be updated to revise the order in which actions are done